itou/templates/siae_evaluations/includes/criteria_form.html

```diff
@@ -0,0 +1,43 @@
+{% load bootstrap4 %}
+
+<form method="post" action="" role="form" class="js-prevent-multiple-submit">
+
+    {% csrf_token %}
+
+    {% bootstrap_form_errors form_administrative_criteria type="all" %}
+
+    {% if request.user.is_siae_staff %}
+
+        {% if level_1 %}
+            <h3 class="h2">
+                Critères administratifs de niveau 1
+            </h3>
+
+
+            {% for field in form_administrative_criteria %}
+                {% if form_administrative_criteria.LEVEL_1_PREFIX in field.name %}
+                    {% include "eligibility/includes/criteria_input.html" with field=field objects_dict=form_administrative_criteria.OBJECTS %}
+                {% endif %}
+            {% endfor %}
+        {% endif %}
```

Comment:
moui, c'est un peu dommage ces deux boucles pour level1 et level2.
Tu refais ici dans le code de la template, le meme `if` à peu de choses près qui te sert à générer `level_1` dans ton contexte. Pourquoi ne pas filer au contexte justement, dans `level_1` la liste des critères administratifs de niveau 1 et itérer dessus, plutot que juste l'utiliser comme switch et réécrire une condition derrière ?

Reply:
:100:  J'ai préféré séparé les champs du formulaire dans la vue, par facilité. Résultat (à débattre), les champs sont transmis deux fois au template : via le formulaire `form_administrative_criteria` et via les deux listes de champs `level_1_fields` et `level_2_fields`. Ça me parait bizarre